services/orchestrator-agent/webhook_server.py

```
@@ -186,14 +187,60 @@ async def rbac_org_members(request: web.Request) -> web.Response:
+        return web.json_response({"error": f"unknown role: {role_name}"}, status=404)
+    if not rbac_engine.delete_role(role_name):
+        return web.json_response({"error": "cannot delete built-in role"}, status=400)
+    await rbac_store.delete_role(role_name)
+    return web.json_response({"deleted": role_name})
+
+
+async def rbac_membership_delete(request: web.Request) -> web.Response:
+    """Revoke a membership (role assignment)."""
+    try:
+        body = await request.json()
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)